library/src/test/java/com/chuckerteam/chucker/internal/support/ShareUtilsTest.kt

```diff
@@ -0,0 +1,56 @@
+package com.chuckerteam.chucker.internal.support
+
+import android.content.Context
+import com.chuckerteam.chucker.R
+import com.chuckerteam.chucker.TestTransactionFactory
+import com.chuckerteam.chucker.internal.data.entity.HttpTransaction
+import io.mockk.every
+import io.mockk.mockk
+import kotlin.random.Random
+import org.junit.Test
+
+class ShareUtilsTest {
+
+    private val contextMock = mockk<Context> {
+        every { getString(R.string.chucker_url) } returns "URL"
+        every { getString(R.string.chucker_method) } returns "Method"
+        every { getString(R.string.chucker_protocol) } returns "Protocol"
+        every { getString(R.string.chucker_status) } returns "Status"
+        every { getString(R.string.chucker_response) } returns "Response"
+        every { getString(R.string.chucker_ssl) } returns "SSL"
+        every { getString(R.string.chucker_yes) } returns "Yes"
+        every { getString(R.string.chucker_no) } returns "No"
+        every { getString(R.string.chucker_request_time) } returns "Request time"
+        every { getString(R.string.chucker_response_time) } returns "Response time"
+        every { getString(R.string.chucker_duration) } returns "Duration"
+        every { getString(R.string.chucker_request_size) } returns "Request size"
+        every { getString(R.string.chucker_response_size) } returns "Response size"
+        every { getString(R.string.chucker_total_size) } returns "Total size"
+        every { getString(R.string.chucker_request) } returns "Request"
+        every { getString(R.string.chucker_body_omitted) } returns "(encoded or binary body omitted)"
+        every { getString(R.string.chucker_export_separator) } returns "=================="
+        every { getString(R.string.chucker_export_prefix) } returns "/* Export Start */"
+        every { getString(R.string.chucker_export_postfix) } returns "/*  Export End  */"
+    }
+
+    @Test
+    fun isStringFromTransactionsListValid() {
+        val transactionList = ArrayList<HttpTransaction>()
+
+        for (i in 0 until 10)
+            transactionList.add(TestTransactionFactory.createTransaction(getRandomHttpMethod()))
+
+        val actualStringFromTransactions = ShareUtils.getStringFromTransactions(transactionList, contextMock)
+        val expectedStringFromTransactions = transactionList.joinToString(
+            separator = "\n${contextMock.getString(R.string.chucker_export_separator)}\n",
+            prefix = "${contextMock.getString(R.string.chucker_export_prefix)}\n",
+            postfix = "\n${contextMock.getString(R.string.chucker_export_postfix)}\n"
```

Comment:
Here you're constructing the expected string, you don't need to call `contextMock.getString` but just copy over the string.

Reply:
updated